### ISSUE TITLE
Fix Label's close button aria-label

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#6474]
**Describe the issue. What is the expected and unexpected behavior?**
The Label component has a weird `aria-label` for the close button. It's currently "label-close-button" which should be updated to be something more accessible.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

https://www.patternfly.org/v4/components/label
